pkg/metrics/metrics.jsx

```diff
@@ -1667,33 +1676,68 @@ class MetricsHistory extends React.Component {
             );
         }
 
+        const columnVisibilityMenuItems = this.columns.map(itm => {
+            return (
+                <SelectOption
+                    key={itm[0]}
+                    value={itm[1]}
+                    inputId={'column-visibility-option-' + itm[0]} />
+            );
+        });
+        const selections = (
+            this.columns
+                    .filter(itm => this.state.selectedVisibility[itm[0]])
+                    .map(itm => itm[1])
+        );
+        const graphsVisibilitySelector = (
+            <Select
+                toggleAriaLabel={_("Graph visibility options menu")}
+                className="metrics-label"
+                variant={SelectVariant.checkbox}
+                isCheckboxSelectionBadgeHidden
+                isOpen={!!this.state.isOpenColumnVisibility}
+                onSelect={(_, selection) => {
+                    const s = this.columns.find(itm => itm[1] == selection);
+                    this.setState({ selectedVisibility: { ...this.state.selectedVisibility, [s[0]]: !this.state.selectedVisibility[s[0]] } });
+                }}
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18299-20230208-164058-692abce6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)